crates/sage-cli/src/input.rs

```diff
@@ -64,6 +68,7 @@ pub struct Input {
 
     annotate_matches: Option<bool>,
     write_pin: Option<bool>,
+    score_type: Option<String>,
```

Comment:
This should be `ScoreType`. Serde will handle deserializing this for us (see how the TMT enum, or tolerance enums are handled).

Reply:
Replaced it accordingly, hope its correct now.